thirdpartynotices.txt

```diff
@@ -32,13 +32,13 @@ THE SOFTWARE.
 
 Pedro Dias - @pedromdias
 
-Licensed under the Apache license, version 2.0 (the "license"); You may not use 
+Licensed under the Apache license, version 2.0 (the "license"); You may not use
 this file except in compliance with the license. You may obtain a copy of the license at:
 
 http://www.apache.org/licenses/LICENSE-2.0.html
-Unless required by applicable law or agreed to in writing, software distributed 
-under the license is distributed on an "as is" basis, without warranties or conditions 
-of any kind, either express or implied. See the license for the specific language 
+Unless required by applicable law or agreed to in writing, software distributed
+under the license is distributed on an "as is" basis, without warranties or conditions
+of any kind, either express or implied. See the license for the specific language
 governing permissions and limitations under the license.
 
 3. dockerfile-language-server-nodejs version 0.0.6 (https://github.com/rcjsuen/dockerfile-language-server-nodejs)
```

Comment:
You might want to update this one. Or maybe make it more generic? Dunno...

Reply:
Good point, thanks.  Removed the versions, don't see them in our other extensions.